UDPBroadcast/UDPBroadcastConnection.swift

```diff
@@ -150,11 +267,81 @@ open class UDPBroadcastConnection: UDPBroadcastSocketCreating {
     /// - Throws: Throws a `ConnectionError` if an error occurs.
     open func sendBroadcast(message: String) throws {
         guard let data = message.data(using: .utf8) else { throw ConnectionError.messageEncodingFailed }
-        try sendBroadcast(data: data)
+        switch addressFamily {
+        case .ipv4:
+            try sendBroadcastv4(data: data)
+            break
+        case .ipv6:
+            try sendBroadcastv6(data: data)
+        }
     }
     
-    open func sendBroadcast(data: Data) throws {
-        fatalError("Must be implemended in a subclass.")
+    /// Send broadcast data.
+    ///
+    /// - Parameter data: Data to send via broadcast.
+    /// - Throws: Throws a `ConnectionError` if an error occurs.
+    private func sendBroadcastv4(data: Data) throws {
```

Comment:
Aren't those two functions here basically the same, apart from `v4address` and `v6address`?

Reply:
yes, I tried to untangle this yesterday with Luizb and failed. We can retry to reduce the redundancy later.